### DISCUSSION TITLE
Производятся работы по переносу сайта

### DISCUSSION THREAD

**Discussion by audetv** [terratensor/svodd#190] [Объявления]
Сайт недоступен. Примерный срок работ до с 9:20 - 10:00 МСК

**Comment by audetv** [comment]
Работы по переносу сайта частично завершены.
Производится настройка сервисов url-shortener, redirect

**Comment by audetv** [comment]
Все сервисы работают в обычном режиме, кроме redirect с fct-search.audetv.ru.
Перенаправление заработает в течение часа после обновления записи DNS. (до 11:30 МСК)

**Comment by audetv** [comment]
Все сервисы работают в обычном режиме. Перенос завершён.

**Reply by iprst** [reply]
То есть теперь все блоки в одном месте расположены?

**Reply by audetv** [reply]
Да, сайт в одном ЦОД территориально.

**Reply by audetv** [reply]
По идее теперь не должно быть подобных случаев, когда где-то на канале между блоками сайта атака, и сайт тормозит. Если только в самом ЦОД проблемы.

**Reply by iprst** [reply]
Да, так будет надёжнее.
